### PR TITLE
test(influxdb): increase influxdb bridge/connector coverage (ee5.0)

### PR DESCRIPTION
**Pull request by thalesmg**
Fixes EMQX-7921

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [x] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/` dir
- [ ] For EMQX 4.x: `appup` files updated (execute `scripts/update-appup.sh emqx`)
- [x] For internal contributor: there is a jira ticket to track this change
- [ ] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

### CODE REVIEW

**Comment by thalesmg** [comment]
- [ ] TODO: Censor `Client` somehow?

```
2022-11-04T15:10:38.034244-03:00 
    client: #{headers =>
                  [{<<"Authorization">>,<<"Token abcdefg">>},
                   {<<"Content-type">>,<<"text/plain; charset=utf-8">>}],
              path => "/api/v2/write?precision=ns&bucket=mqtt&org=emqx",
              pool =>
                  'bridge:influxdb_api_v2:emqx_ee_bridge_influxdb_SUITE:87',
              pool_type => random,protocol => http,version => v2}
    connector: <<"bridge:influxdb_api_v2:emqx_ee_bridge_influxdb_SUITE:87">>
    msg: starting influxdb connector failed
    reason: client is not alive
```

**Comment by thalesmg** [comment]
- [ ] TODO: censor stacktrace 

```
2022-11-04T15:59:21.050597-03:00 
    connector: <<"bridge:influxdb_api_v1:emqx_ee_bridge_influxdb_SUITE:18">>
    error: error
    msg: start influxdb connector error
    reason: boom
    stack: [{emqx_ee_bridge_influxdb_SUITE,'-t_start_exception/1-fun-6-',1,
                [{file,
                     "/home/thales/dev/emqx/emqx/lib-ee/emqx_ee_bridge/test/emqx_ee_bridge_influxdb_SUITE.erl"},
                 {line,804}]},
            {emqx_ee_connector_influxdb,do_start_client,3,
                [{file,
                     "/home/thales/dev/emqx/emqx/lib-ee/emqx_ee_connector/src/emqx_ee_connector_influxdb.erl"},
                 {line,217}]},
            {emqx_ee_connector_influxdb,start_client,2,
                [{file,
                     "/home/thales/dev/emqx/emqx/lib-ee/emqx_ee_connector/src/emqx_ee_connector_influxdb.erl"},
                 {line,196}]},
            {emqx_resource,call_start,3,
                [{file,
                     "/home/thales/dev/emqx/emqx/apps/emqx_resource/src/emqx_resource.erl"},
                 {line,360}]},
            {emqx_resource_manager,start_resource,2,
                [{file,
                     "/home/thales/dev/emqx/emqx/apps/emqx_resource/src/emqx_resource_manager.erl"},
                 {line,488}]},
            {gen_statem,loop_state_callback,11,
                [{file,"gen_statem.erl"},{line,1205}]},
            {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]},
            {influxdb,start_client,
                [[{host,"localhost"},
                  {port,8086},
                  {pool_size,24},
                  {pool,
                      'bridge:influxdb_api_v1:emqx_ee_bridge_influxdb_SUITE:18'},
                  {precision,<<"ns">>},
                  {protocol,http},
                  {version,v1},
                  {username,"root"},
                  {password,"emqx@123"},
                  {database,"mqtt"},
                  {https_enabled,false}]],
                []}]

```

File: apps/emqx/test/emqx_common_test_helpers.erl [hunk 1]
```diff
@@ -769,3 +778,129 @@ expand_node_specs(Specs, CommonOpts) ->
...
+timeout_proxy(on, Name, ProxyHost, ProxyPort) ->
+    Url =
+        "http://" ++ ProxyHost ++ ":" ++ integer_to_list(ProxyPort) ++ "/proxies/" ++ Name ++
+            "/toxics",
+    NameBin = list_to_binary(Name),
+    Body =
+        <<"{\"name\":\"", NameBin/binary,
+            "_timeout\",\"type\":\"timeout\","
+            "\"stream\":\"upstream\",\"toxicity\":1.0,"
+            "\"attributes\":{\"timeout\":0}}">>,
```
[the remark this answers is not in the document]
Comment (thalesmg) [reply apps/emqx/test/emqx_common_test_helpers.erl:859]: Good point; where I first wrote this, there was no JSON encoding available :joy_cat:  :see_no_evil:

[the remark this answers is not in the document]
Comment (thalesmg) [reply apps/emqx/test/emqx_common_test_helpers.erl]: :heavy_check_mark:

**Review by zmstone — APPROVED** [review]

**Review by ieQu1 — APPROVED** [review]

**Pull request merged**